wagtail/images/models.py

```diff
@@ -1372,6 +1372,15 @@ def img_tag(self, extra_attributes={}):
     def __html__(self):
         return self.img_tag()
 
+    def get_mime_type(self):
+        """
+        Returns the MIME type of the rendition file.
+        """
+        # Without a pre-populated field value to return, the most reliable solution
+        # is to use willow to inspect the file
+        with self.get_willow_image() as willow_image:
+            return willow_image.mime_type
```

Comment:
NOTE: I very much suspect that we could come up with a more efficient solution here (populating a field when generating), but that's out-of-scope for this PR